code/style.css

```diff
@@ -1,5 +1,191 @@
-/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Condensed:wght@300;400;700&display=swap');
 
 body {
-  background-color: lightblue;
+  background-color: lightslategray;
+}
+
+video {
+  width: 100%;
+  height: 50%;
+
+}
+
+
+h1.hovedoverskrift {
+  font-family: 'Arya', sans-serif;
+  font-weight: bold;
+  color: antiquewhite;
+  font-size: 80px;
+  text-align: center;
+  margin-top: 1%;
+}
+
+h2.overskrift-2  {
+  font-family: 'Arya', sans-serif;
+  color: antiquewhite;
+  font-weight: normal;
+  font-size: 40px;
+  margin-top: 2%;
+  text-align: center;
+}
+
+p.ingress {
+  font-family: 'Arya', sans-serif;
+  color: antiquewhite;
+  font-style:italic ;
+  font-size: 25px;
+  margin-left: 15%;
+  margin-right: 15%;
+}
+
+
+
+input[type=text] {
+  width: 100%;
+  padding: 12px 20px;
+  margin-top: 2%;
+  margin-bottom: 2%;
+  display: inline-block;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+input[type=submit] {
+  font-family: 'Arya', sans-serif;
+  font-size: 20px;
+  font-weight: bold;
+  color: antiquewhite;
+  width: 100%;
+  margin-top: 10%;
+  background-color: rgb(52, 146, 140);
+  color: whitgreye;
+  padding: 14px 20px;
+  margin: 8px 0;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+}
+
+input[type=submit]:hover {
+  background-color: lightslategrey;
+}
+
+.skjema  {
+  width: 60%;
+  margin-top: 3%;
+  margin-left: 18%;
+  font-family: 'Arya', sans-serif;
+  font-size: 20px;
+  color: grey;
+  border-radius: 5px;
+  background-color: whitesmoke;
+  padding: 20px;
+}
+
+::-webkit-input-placeholder {
+  font-family: 'Arya', sans-serif;
+  font-size: 15px;
+  color: lightgray;
+}
+
+select {
+  padding: 10px;
+  margin-top: 3%;
+  margin-bottom: 3%;
+  font-family: 'Arya', sans-serif;
+  font-size: 15px;
+  color:grey;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box;
+  width: 100%;
+  display: inline-block;
+}
+
+input {
+  width: 100%;
+}
+
+* {box-sizing: border-box;
+}
```

Comment:
Since this applies to everything, you should have it in the top. Try to structure your CSS in the same order that the HTML elements are ordered